src/app/pages/private/collection/collection.page.ts

```diff
@@ -87,23 +89,36 @@ export default class CollectionPage implements OnInit {
   collectionInfoError: Signal<string | undefined> = this.collectionInfoStore.error;
   isCollectionInfoLoading: Signal<boolean> = this.collectionInfoStore.loading;
 
-  isCollectionDataLoadedSuccessfuly: Signal<boolean> = computed(() => {
-    const isLoaded = this.isCollectionInfoLoading() === false;
-    const hasNoError = this.collectionInfoError() === undefined;
+  collectionHasNoError: Signal<boolean> = computed(() => {
```

Comment:
it looks like we can drop collectionHasNoError and isCollectionInfoLoaded and directly compare with wanted values in template. It will reduce amount of needed code and make it mroe readable, since condition if (!collectionHasNoError) looks a bit hard to understand from human perspective